.gitignore

```diff
@@ -12,7 +12,12 @@ _version.pyi
 
 # Documentation ignores
 docs/_build/
+docs/_static/
```

Comment:
This should definitely not be here as this is where one stores any static images etc.

Reply:
I'll amend the commit and remove that line.